src/containerapp/azext_containerapp/_params.py

```diff
@@ -299,6 +302,14 @@ def load_arguments(self, _):
     with self.argument_context('containerapp ingress sticky-sessions') as c:
         c.argument('affinity', arg_type=get_enum_type(['sticky', 'none']), help='Whether the affinity for the container app is Sticky or None.')
 
+    with self.argument_context('containerapp ingress cors') as c:
+        c.argument('allowed_origins', nargs='*', options_list=['--allowed-origins', '-r'], help="A list of allowed origin(s) for the container app. Values are space-separated. If you want to clear it you can use --allowed-origins with no arguments.")
+        c.argument('allowed_methods', nargs='*', options_list=['--allowed-methods', '-m'], help="A list of allowed method(s) for the container app. Values are space-separated. If you want to clear it you can use --allowed-methods with no arguments.")
+        c.argument('allowed_headers', nargs='*', options_list=['--allowed-headers', '-a'], help="A list of allowed header(s) for the container app. Values are space-separated. If you want to clear it you can use --allowed-headers with no arguments.")
```

Comment:
https://github.com/Azure/azure-cli-extensions/blob/d54edd38bb1551bec9a4d9e81dee4a5377ea7e96/src/containerapp/azext_containerapp/_params.py#LL70C1-L72C183

> Empty string to clear existing values

Could the instructions for clearing existing values be consistent in the help message?

Reply:
Fixed